scripts/34-Plain-Text-Team.md

```diff
@@ -0,0 +1,601 @@
+
+GitHub formats Markdown very nicely, but you can export it in any format you like, and edit it with any tool you like, present and future.
+
+Including my favourite tool here, Obsidian.
+
+Markdown keeps your team focussed on what is important by allowing you just enough formatting, but no rich customisation options.
+You're not making a beautiful client brochure, so you shouldn't use 90s desktop publishing tools to make your company's critical documentation.
+Should you?
+
+Back to GitHub's features:
+
+---
+
+![[issues.png]]
+
+## Issues
+
+notes:
+
+Though not part of the git system, Issues are a natural addition that all git hosts have implemented:
+A simple task system for capturing work that needs doing.
+These could be new features, customer requests, bug reports, or ideas.
+They have a rich comment thread for discussion, can be assigned to team members, and tagged with custom tags.
+
+This minimum viable project management system could be all you need.
+Certainly, for a solo or small team, capturing requirements in Issues might be enough.
+
+But if you need more, you need Milestones.
+
+---
+![[milestones.png]]
+
+## Milestones
+
+notes:
+GH milestones are a grouping of issues with a deadline.
+
+They typically represent a target, a release, or something the team is working towards.
+Milestones just have a title, a description, and a date.
+No burndown charts, no swim lanes, no complex statistics, just a progress bar.
+
+This might be enough project management for you.
+If not, it is time for GH Projects.
+
+---
+![[projects.png]]
+
+## Projects
+
+notes:
+
+Each GitHub repo, team, and organisation can have a project board, a lightweight kanban board with customisable columns, allowing you to group your issues together, and observe their progress through your current iteration.
+
+This represents the information radiator for your team, a bird's-eye view of what is happening with the project, and something you might gather around for your morning catchup meeting.
+
+You don't need all the features of Trello, JIRA, or anything like that. You need the minimum viable board.
+
+This bare-bones tool completely side-steps 'the JIRA effect', which is if you have a tool that is packed full of time tracking, velocity points, and so on, the temptation is to use all these features, even if they give no value, and complicate your processes.
+
+---
+
+## Aside
+
+# Standups Are Great
+
+notes:
+
+If you have one meeting a day, it should be a standup.
+Good standups replace other meetings and accelerate your project dramatically.
+
+The way I like to do standups is not by asking everyone what they did yesterday and what they intend to do today.
+
+That's a great way to find out at length what Dave did on his day off, but not a good way to find out what's happening with the project specifically.
+
+I recommend walking your board, backwards, from right to left.
```

Comment:
Did you get this technique from anywhere? I think it may add credence and respectability to your point if you could reference some giants that you're building on the shoulders of. Just a thought.

Reply:
(In fact, in creating Projects, GitHub directly put a few other companies out of business)

Reply:
don't recall, just the way the best teams I've worked with did it. it's simple enough that surely it's common

Reply:
Yes. That's fine, I just thought the point would be strengthened if you had any other writing you could reference. But if you don't know of any, leave it as is.